lte/gateway/python/magma/pipelined/app/ng_services.py

```diff
@@ -0,0 +1,80 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from .base import MagmaController, ControllerType
+from magma.pipelined.ng_manager.node_state_manager import NodeStateManager
+from magma.common.service_registry import ServiceRegistry
+
+SESSION_SERVICE_NAME = "sessiond"
+
+class NGServiceController(MagmaController):
+    """
+    This class is intended to be a place holder for
+    all 5G services which include node and session
+    management
+    """
+    APP_NAME = "ng_services"
+    APP_TYPE = ControllerType.LOGICAL
+
+    def __init__(self, *args, **kwargs):
+        super(NGServiceController, self).__init__(*args, **kwargs)
+
+        # General Initialization
+        self.loop = kwargs['loop']
+        self._datapath = None
+        self.config = kwargs['config']
+
+        #Get SessionD Channel
+        self.sessiond_chan = \
+             ServiceRegistry.get_rpc_channel(SESSION_SERVICE_NAME, ServiceRegistry.LOCAL)
+
+        # Initialize ng services
+        self._ng_node_mgr = NodeStateManager(self.loop, self.sessiond_chan, self.config)
+        self._ng_sess_mgr = None
+
+        # Cleanup actions
+        self._clean_restart = kwargs['config']['clean_restart']
+
+    def initialize_on_connect(self, datapath):
```

Comment:
Why are we maintaining a datapath var in this class? I don't think it has any purpose. Can we remove it completely, as well as _clean_restart

Reply:
Addressed the comments.

Reply:
hm its still here

Reply:
Somehow it was not came in the view I have. But re-checked to ensure its removed

Reply:
having `datapath` unused arg might tick off pep python style check. just do `initialize_on_connect(self, _):`